migrations/20220201085056_pulling_data/migration.sql

```diff
@@ -0,0 +1 @@
+-- This is an empty migration.
```

Comment:
Looks like we don't need this migration

Reply:
Thanks for the review ! I will fix the mentioned problems am update the pull request ASAP :)